### PR TITLE
changing dockers to apt update latest

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

**Comment by rafie** [comment]
@chayim LGTM. Is this for security patches?

**Comment by chayim** [comment]
Yeah, by doing the default apt up, we get whatever is in the feeds.

**Review by rafie — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -32,6 +32,10 @@ ADD . /build
 
 RUN ./deps/readies/bin/getpy2
 RUN ./system-setup.py
+RUN apt-get update -qq
```
Comment (rafie) [inline Dockerfile:35]: @chayim This won't work on OSes that are not Debian-like. We better have something going for us within `system-sytem.py`.

Comment (chayim) [reply Dockerfile:35]: Yeah, I just remembered that there are non-Ubuntu dockers... Though currently random. I'll re-inject support into Dockerfiles for updates via apt or yum for now. This isn't something I'd like system-setup to do, given that it also gets used on bare metal occasionally (heck, by myself).

Comment (rafie) [reply Dockerfile]: I agree this is not `system-setup.py`'s role in life (though it's bound to do `apt-get update` to be able to install anything at all). We probably need to have this as a standard procedure in our Dockerfiles, just abstracted per package manager. So, how about something like `readies/bin/getget`, based on `is_command`, and simply performing the `apt-get upgrade` thing for each package manager. Maybe call it `getupdates`?

**Comment by rafie** [comment]
Handled by #571, can close.

**Pull request closed without merging**